### PR TITLE
[data ingestion] initial draft version of the docs

### PR DESCRIPTION
**Pull request by phoenix-o**

### CODE REVIEW

**Comment by phoenix-o** [comment]
@ronny-mysten:  just curious, why operator section is preferred? it's expected for developers to write some code to actually use this framework

File: docs/content/guides/operator/data-ingestion.mdx [hunk 1]
```diff
@@ -0,0 +1,100 @@
+---
+title: Custom Indexer
+description: You can build custom indexers using the Sui micro-data ingestion framework. To create an indexer, you subscribe to a checkpoint stream with full checkpoint content. Establishing a custom indexer helps improve latency, allows pruning the data of your Sui Full node, and provides efficient assemblage of checkpoint data.
+---
+
+You can build custom indexers using the Sui micro-data ingestion framework. To create an indexer, you subscribe to a checkpoint stream with full checkpoint content. This stream can be one of the publicly available streams from Mysten Labs, one that you set up in your local environment, or a combination of the two.  
+
+![Data ingestion flow](./images/data-ingestion.png)
```
Comment (phoenix-o) [inline docs/content/guides/operator/data-ingestion.mdx:8]: I would prefer not to include this version of a diagram(it's a bit mysten specific). If you think it's necessary, I'll redo the diagrams and move it to a different place on this page

Comment (ronny-mysten) [reply docs/content/guides/operator/data-ingestion.mdx:8]: That would be awesome. I was going to say that I would do it, but you'd make a better one.

**Comment by ronny-mysten** [comment]
> @ronny-mysten: just curious, why operator section is preferred? it's expected for developers to write some code to actually use this framework

First, anything I remark on is only a suggestion. Feel free to override me (like the change of title). This is kind of a gray area as it's for developer's that are dealing with a full node. If we stick it in the developer section, I don't know that it should be in getting started. I'd probably say Advanced Topics unless you think most folks will need to use this?

**Review by ronny-mysten — APPROVED** [review]
Let me know if I need to change anything in the image. Or move it. Change anything you want. Let me know if custom indexer name isn't cool. Otherwise, lgtm.

**Comment by phoenix-o** [comment]
that's awesome! thanks a lot @ronny-mysten!! I'll take a final look/iteration early next week

**Pull request merged**